arreglo/src/main/java/com/globant/model/Payment.java

```diff
@@ -0,0 +1,37 @@
+package com.globant.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "payments")
+public class Payment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+    private float amount;
+
```

Comment:
Payment should have a @OneToOne relationship with Order, also declare a @JoinColumn with a foreignKey definition and a referenced column name